packages/facility-server/app/routes/apiv1/appointments.js

```diff
@@ -99,15 +99,17 @@ appointments.post(
       const { email } = appointmentData;
       if (email) {
         await sendAppointmentReminder({
-          appointmentId: result.id,
+          appointmentId: appointment.id,
           email,
           facilityId,
           models,
           settings,
         });
       }
-      res.status(201).send(result);
```

Comment:
I believe that calling `res.send()` inside the transaction block was causing this issue, as it's possible that we respond to the request before we actually write the data to the database